### ISSUE TITLE
The method should be platform-agnostic, not the code

### ISSUE THREAD

**Issue by anqur** [lingzerol/zeropy#1]
https://github.com/lingzerol/zeropy/blob/ddc84b42f9fac2435b8fff4896c1e1b36d883e61/common/input.py#L8

Platform-agnostic is the method `input_with_timeout`, not the code, hence:

```py
def input_with_timeout():
    # Doing the platform-specific logic...
    if platform.system() == "Windows":
        # ...
    elif ...:
        # ...
```

**Comment by lingzerol** [comment]
Right, I think the user should provide the "timer" parameter on windows at the beginning, and now it is ok to delete the "timer" to prevent from existing two different method.

**Comment by anqur** [comment]
@lingzerol You can reference this issue in your commit message, e.g.

* `git commit -m "ref #1"` to reference issue No.1
* `git commit -m "fix #1` to reference issue No.1 and close it

Please make sure your commit is related to a specific issue, and close it by your commit if neccessary, since it's a good habit.

**Comment by lingzerol** [comment]
ok.[dog head]
